src/main/java/com/example/demo/service/BookService.java

```diff
@@ -0,0 +1,10 @@
+package com.example.demo.service;
+
+import com.example.demo.model.Book;
+import java.util.List;
+
+public interface BookService {
+    Book save(Book book);
+
+    List findAll();
```

Comment:
It is worth remembering the topic of generics
```suggestion
    List<Book> findAll();
```